### PR TITLE
Add Lisp team as maintainers of Lisp packages

### PR DESCRIPTION
**Pull request by Uthar**
Set the default value of the `meta.maintainers` attribute to the Lisp team for all packages in the Lisp ecosystem. Is this what we want?

This will apply to all "libraries" i.e. `sbclPackages.*`. "Programs" such as stumpwm or Nyxt, or the Lisp compilers themselves, already have maintainers

See #218870

### CODE REVIEW

**Comment by 7c6f434c** [comment]
Will it work as desired when `args` already has `meta.hydraPlatforms`?

If we want the entire team to have visibility, I would be in favour of replacing myself with the team for the Common Lisp implementations (and whatever Common-Lisp-adjacent you find me maintaining and consider in scope).

**Comment by Uthar** [comment]
> Will it work as desired when `args` already has `meta.hydraPlatforms`?

hmm right, it gets overridden

**Comment by Uthar** [comment]
Both `hydraPlatform` and `maintainers` work now:  02cc867

**Comment by Uthar** [comment]
Ok , I'll set the team in CL implementations too

File: pkgs/development/lisp-modules/nix-cl.nix
Comment (7c6f434c) [inline pkgs/development/lisp-modules/nix-cl.nix:225]: Hmm, this doesn't allow to have extra maintainers for e.g. Nyxt, right?

**Comment by nagy** [comment]
I am happy to help maintaining lisp packages too, if you want.

File: pkgs/development/lisp-modules/nix-cl.nix
Comment (Uthar) [reply pkgs/development/lisp-modules/nix-cl.nix:225]: Currently nyxt doesn't have this problem, because it's made from a separate `mkDerivation` call, and uses `sbclPackages.nyxt` only as a "source". Stumpwm does have this problem, because it comes directly from `sbclPackages`, but there we could use `overrideAttrs` to change `meta.maintainers`.

Comment (7c6f434c) [reply pkgs/development/lisp-modules/nix-cl.nix:225]: Maybe just say `maintainers = args.meta.maintainers or lib.teams.lisp.members` ? (alternatively, `or []` and append the team)

**Review by RaitoBezarius — APPROVED** [review]
Awesome <3.
LGTM; please let me know when you want a merge, if needed. :)

File: pkgs/development/lisp-modules/nix-cl.nix
Comment (Uthar) [reply pkgs/development/lisp-modules/nix-cl.nix]: Ok 6817b72ca3cdb8e5edd9ff84a1dc549de1c683f3

**Pull request merged**
